earth_enterprise/src/maps/bootstrap.js

```diff
@@ -1,32 +0,0 @@
-/*
```

Comment:
I've seen web apps that use these two boostrap files. Are they causing problems? If not, it might be wise to keep them.

Reply:
I see the bigger picture now: these are easy entry points for users who want to override the api version with a specific build's subfolder.  I'll add them back.

Reply:
...and after discussing it with you a bit more, we concluded that it's best to check this PR in as-is, and potentially re-add the versioned API bootstrapper(s) when the alternate versions are re-introduced to the codebase.